app/assets/stylesheets/base.scss

```diff
@@ -0,0 +1,83 @@
+:root {
+  --active-tab-color: blue;
+  --text-color: black;
+  --btn-background: #79d733;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  background: #fbfbfb;
+}
+
+nav {
+  height: 68px;
+  width: 100%;
+  background: #ffffff;
+  box-shadow: 0 2px 4px #6596ff26;
+  padding: 12px 55px 12px 0;
+
+  .circle-button {
+    border-radius: 50%;
+    height: 38px;
+    width: 38px;
+    background: var(--btn-background);
+    border: 0 solid var(--btn-background);
+    font-size: 20px;
+    box-shadow: 0 0 3px 2px #6596ff26;
+  }
+
+  .outer-border {
+    height: 44px;
+    border-radius: 50%;
+    padding: 2px;
+    width: 44px;
+    font-size: 20px;
+    border: 1px solid #000000;
+  }
+
+  .email {
+    font-size: 14px;
+  }
+
+  .icon-logout {
+    color: #c4183c;
+    padding-top: 13px;
+  }
+
+  .logout {
+    background: #ffffff;
+    box-shadow: 0 2px 10px #6596ff26;
+    border: 0.5px solid #afafaf;
+    border-radius: 4px;
+  }
+
+  .sign-out {
+    text-align: left;
+    font-family: Avenir;
+    font-size: 14px;
+    letter-spacing: 0;
+    color: #c4183c;
+    padding: 15px 0 0 0;
+    text-transform: capitalize;
+  }
+}
+
+a:hover{
+  text-decoration: none;
+}
+
+.red {
+  color: red;
+}
+
+.blue {
+  color: blue;
+}
+
+.bg-color {
```

Comment:
We can change class name, so that when we use it we will know that it will have default background color.?
What do you suggest?
I would suggest `app-background` pr `app-bg`